src/Kentico.Xperience.AzureSearch/AzureSearchStartupExtensions.cs

```diff
@@ -120,8 +120,7 @@ internal class AzureSearchBuilder : IAzureSearchBuilder
 
     private void ValidateIndexSearchModelProperties<TSearchModel>() where TSearchModel : IAzureSearchModel, new()
     {
-        var type = typeof(TSearchModel)
-            ?? throw new InvalidOperationException(ErrorMessage);
+        var type = typeof(TSearchModel);
```

Comment:
A `typeof()` should never be `null` because the generic type will always exist if the code compiles.